docs/tutorials/lut.rst

```diff
@@ -3,29 +3,31 @@ The traveltime lookup table
+First, we choose a pair of coordinate reference systems to represent the input coordinate space (``cproj``) and the Cartesian grid space (``gproj``). We do this using `pyproj`, which provides the Python bindings for the PROJ library. It is important to think about which projection is best suited to your particular study region. More information can be found [in their documentation](https://pyproj4.github.io/pyproj/stable/).
 
-We use here the WGS84 reference ellipsoid (used as standard by the Global Positioning System) as our input space and the Lambert Conformal Conic projection to form our Cartesian space. The units of the Cartesian space are specified as metres. The values used in the LCC projection are for a study region in Sabah, Borneo.
+.. warning:: The default units of :class:`Proj` are `metres`! It is strongly advised that you explicitly state which units you wish to use.
+
+We use here the WGS84 reference ellipsoid (used as standard by the Global Positioning System) as our input space and the Lambert Conformal Conic projection to form our Cartesian space. The units of the Cartesian space are specified as kilometres. The values used in the LCC projection are for a study region in Sabah, Borneo.
 
 ::
 
 	from pyproj import Proj
 
 	cproj = Proj(proj="longlat", ellps="WGS84", datum"=WGS84", no_defs=True)
 	gproj = Proj(proj="lcc", lon_0=116.75, lat_0=6.25, lat_1=4.0, lat_2=7.5,
-	         datum="WGS84", ellps="WGS84", units="m", no_defs=True)
+	         datum="WGS84", ellps="WGS84", units="km", no_defs=True)
 
 Geographical location and spatial extent
 ########################################
-In order to geographically situate our lookup table, we choose two reference points in the input coordinate space, herein called the lower-left and upper-right corners (``ll_corner`` and ``ur_corner``, respectively). By default, we work in a depth-positive frame (i.e. positive-down or left-handed coordinate system) and use units of metres. It is possible to run QuakeMigrate with distances measured in metres, as long as the user specifies this requirement when defining the grid projection and all inputs (station elevations, grid specification, etc) are in metres.
+In order to geographically situate our lookup table, we choose two reference points in the input coordinate space, herein called the lower-left and upper-right corners (``ll_corner`` and ``ur_corner``, respectively). By default, we work in a depth-positive frame (i.e. positive-down or left-handed coordinate system) and use units of kilometres. It is possible to run QuakeMigrate with distances measured in metres, as long as the user specifies this requirement when defining the grid projection and all inputs (station elevations, grid specification, etc) are in metres.
```

Comment:
I do think it’s important to mention here that seismic velocities also need to be in consistent units. Currently this point isn’t made in the tutorial (I don’t think - apologies if I’ve missed it).